Squablr/View Controllers/UserFeedViewController.m

```diff
@@ -89,6 +67,13 @@ - (void)handleReload:(UIBarButtonItem *)sender {
 - (void)swipeableView:(ZLSwipeableView *)swipeableView
          didSwipeView:(UIView *)view
           inDirection:(ZLSwipeableViewDirection)direction {
+    // Handle direction
```

Comment:
What is this code doing?

Reply:
its blank but will be handling what happens when the user swipes left or right ie favorited/favoritee table